templates/mwe.md

```diff
@@ -91,24 +85,25 @@ It is fine for your example to throw compiler errors or warnings. In particular,
 
 ## How do I know if my code is a MWE?
 
-You should *test* this by making a new Lean file, pasting your code snippet into it, and seeing if you get the expected behavior. This is exactly what people who try to help you will do!
+You should *test* this by copy-pasting your code snippet into the [lean web editor](https://lean.math.hhu.de), or into a new Lean file, and seeing if you get the expected behavior. This is exactly what people who try to help you will do!
 
-## What if I'm asking about the [Natural Number Game](https://www.ma.imperial.ac.uk/~buzzard/xena/natural_number_game/)?
+## What if I'm asking about the [Natural Number Game](https://adam.math.hhu.de/)?
```

Comment:
Why not https://adam.math.hhu.de/#/g/hhu-adam/NNG4?

Reply:
I don't have a strong opinion about linking the landing page, Ill change it.